test/integration/admin_dashboard_test.rb

```diff
@@ -499,6 +499,68 @@ def teardown
     assert_equal [prev_thesis_id, h.thesis_id], h.versions.last.changeset[:thesis_id]
   end
 
+  test 'can identify the user who created a hold' do
+    creator = users(:thesis_admin)
+    mock_auth(creator)
+
+    orig_count = Hold.count
+
+    post admin_holds_path,
+      params: { hold: {       
+                        thesis_id: Thesis.first.id, 
+                        hold_source_id: HoldSource.first.id,
+                        date_requested: '2021-03-02',
+                        date_start: '2021-03-02',
+                        date_end:'2021-03-02',
+                        case_number: nil,
+                        status: 'active',
+                        processing_notes: nil
+                      },
+              }
+    assert_equal orig_count + 1, Hold.count
+
+    hold = Hold.last
+    hold_creator = User.find_by(id: hold.versions.first.whodunnit)
+    assert_equal hold_creator.id, creator.id
+    assert_equal hold_creator.kerberos_id, hold.created_by
+  end
+
```

Comment:
We should add a test for `can handle a user who created a hold that no longer exists` [there's a related comment in the `hold.rb` code]